app/players/elves/elf.py

```diff
@@ -0,0 +1,11 @@
+from app.players.player import Player
+
+
+class Elf(Player):
+    def __init__(self, nickname: str, musical_instrument: str) -> None:
+        super().__init__(nickname)
+        self._musical_instrument = musical_instrument
+
+    def play_elf_song(self) -> None:
+        print(f"{self.nickname} is playing a song on "
+              + f"the {self._musical_instrument}")
```

Comment:
The `play_elf_song` method correctly prints the message using the `nickname` and `_musical_instrument` attributes. Ensure that the `nickname` attribute is inherited and accessible from the `Player` class.